geoconvert/data/countries.py

```diff
@@ -334,6 +340,7 @@
         "bolivia plurinational state of": "BO",
         "bonaire saint eustatius and saba": "BQ",
         "bosnia and herzegovina": "BA",
+        "bosnia & herzegovina": "BA",
```

Comment:
Couldn't we consider that either "bosnia" or "herzegovina" is enough to detect BA ?

Reply:
I'd rather keep it as it is for the moment, but that's a good point. 
I think we should postpone what you are proposing since we might want to do that for other countries in the future. 
It also seems to me that it needs some further thoughts before implementing such a change.